src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -113,8 +115,22 @@ def get_latest_installed(cls):
         AnsysVersionNotFound
             If an Ansys version cannot be found.
         """
+
```

Comment:
@mkundu1

In this class, we can add ~ `get_latest_installed_fluent_exe_path()` implemented in terms of `get_latest_installed()` and adding the new code that maps Ansys path to Fluent path. Then we can return the full path that we are checking below. The calling module for `get_latest_installed()` also does the same Ansys to Fluent path mapping so by adding this method we can eliminate the duplication.